pkgs/games/lbreakout2/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, SDL, SDL_mixer, zlib, libpng, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "lbreakout2-${version}";
+  version = "2.6.5";
+  buildInputs = [ SDL SDL_mixer zlib libpng gcc ];
```

Comment:
gcc is already added to the build environment via stdenv (unless cc is overridden). Is there a particular reason for adding it explicitly?